da/registry/registry_test.go

```diff
@@ -14,7 +17,21 @@ func TestRegistery(t *testing.T) {
 
 	assert.ElementsMatch(expected, actual)
 
-	for _, e := range expected {
+	constructor := func() da.DataAvailabilityLayerClient {
+		return &mock.DataAvailabilityLayerClient{} // cheating, only for tests :D
+	}
+	err := Register("testDA", constructor)
+	assert.NoError(err)
+
+	// re-registration should fail
+	err = Register("celestia", constructor)
```

Comment:
Why should this fail? Isn't the first DA "testDA" and the second "celestia"? Where is celestia already in the registry?

Reply:
[Right now `](https://github.com/rollkit/rollkit/pull/721/files#diff-5aa4149662046f4fc53afe0172f9a2926b79f0850764241f70c09eeef0a5e36eR15)

We have 3 DAs added by default, in `registry.go`.

Reply:
ahh i see makes sense